test/get_test.go

```diff
@@ -43,6 +62,150 @@ func TestGet(t *testing.T) {
 			g.Expect(resp.Kvs[0].Value).To(Equal([]byte("testValue")))
 		}
 	})
+
+	t.Run("KeyRevision", func(t *testing.T) {
+		g := NewWithT(t)
+
+		var lastModRev int64
+
+		// Create a key with a known value
+		{
+			resp, err := client.Txn(ctx).
+				If(clientv3.Compare(clientv3.ModRevision("testKey"), "=", 0)).
+				Then(clientv3.OpPut("testKey", "testValue")).
+				Commit()
+			g.Expect(err).To(BeNil())
+			g.Expect(resp.Succeeded).To(BeTrue())
+			lastModRev = resp.Responses[0].GetResponsePut().Header.Revision
+		}
+
+		// Get the key's version
+		{
+			resp, err := client.Get(ctx, "testKey", clientv3.WithCountOnly())
+			g.Expect(err).To(BeNil())
+			g.Expect(resp.Count).To(Equal(int64(1)))
+		}
+
+		// Update the key
+		{
+			resp, err := client.Txn(ctx).
+				If(clientv3.Compare(clientv3.ModRevision("testKey"), "=", lastModRev)).
+				Then(clientv3.OpPut("testKey", "testValue2")).
+				Else(clientv3.OpGet("testKey", clientv3.WithRange(""))).
+				Commit()
+			g.Expect(err).To(BeNil())
+			g.Expect(resp.Header.Revision).To(Equal(int64(2)))
+		}
+
+		// Get the updated key's version
+		{
+			resp, err := client.Get(ctx, "testKey", clientv3.WithCountOnly())
+			g.Expect(err).To(BeNil())
+			g.Expect(resp.Count).To(Equal(int64(1)))
```

Comment:
This tests that one kv was fetched, but the comment suggests otherwise?

Reply:
Has this been addressed? What exactly do we test here?

Reply:
I was checking the count of entries to be the same as before, but now checking the updated value.